cset-workflow/meta/diagnostics/rose-meta.conf

```
@@ -4,9 +4,9 @@
 
 [template variables]
 
-[template variables=AGGREGATE_ACROSS_CASE_STUDIES]
+[template variables=SHOW_CASE_STUDY_AGGREGATE_OPTIONS]
 ns=Diagnostics/Quicklook
-description=Aggregate over case studies.
+description=Show/hide case study aggregation toggles.
```

```suggestion
description=Show case study aggregation toggles.
```
Given this is a true/false setting, we don't need to say hide.